src/bci_build/package.py

```diff
@@ -1364,6 +1364,23 @@ def _get_openjdk_kwargs(
     )
 ]
 
+PROMETHEUS_CONTAINERS = [
+    ApplicationStackContainer(
+        package_name="prometheus-image",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        name="prometheus",
+        pretty_name="Prometheus",
+        package_list=["golang-github-prometheus-prometheus"],
+        version="2.32.1",
```

Comment:
This is not going to work, this is just the version in Factory, but in SLE we are at 2.18 at the moment.

Please use a replacement for this like it is e.g. done with the postgress container. That uses https://github.com/openSUSE/obs-service-replace_using_package_version in the background.

Reply:
Where does it pull the packages from? We have 2.32.1 in [SLE-15-SP5:GA](https://build.suse.de/package/show/SUSE:SLE-15-SP5:GA/golang-github-prometheus-prometheus).